CHANGELOG.md

```diff
@@ -20,6 +20,26 @@ N/A
 public func setLineHeight(_ lineHeight: CGFloat)
 ```
 
+- CGRectExtension
+
+```swift
+public var x: CGFloat
+public var y: CGFloat
+public func with(x: CGFloat) -> CGRect
+public func with(y: CGFloat) -> CGRect
+public func with(width: CGFloat) -> CGRect
+public func with(height: CGFloat) -> CGRect
```

Comment:
Should we have a `func with(origin: CGPoint) -> CGRect` and `func with(size: CGSize) -> CGRect` or is that too much?